runtime_test.go

```diff
@@ -2168,7 +2323,7 @@ func TestExceptionWithinNativeFunction(t *testing.T) {
 	expected := `TypeError: oh no!
 	at myNativeFunc (native)
 	at myFunc (<eval>:3:3(1))
-	at <eval>:5:8(7)
+	at <eval>:5:8(4)
```

Comment:
I think I asked this in another PR but I wanted to double check these changes made sense, there are a few more.

Reply:
this should be fine